lib/src/realm_object.dart

```diff
@@ -178,7 +179,7 @@ class RealmCoreAccessor implements RealmAccessor {
   }
 }
 
-mixin RealmEntity {
+mixin RealmEntity implements Finalizable {
```

Comment:
Should we implement `Finalizable` on `RealmObject` and add `keepAlive` extension?

Reply:
I believe it is enough to have it on the base `RealmEntity` as `RealmEntityInternal` extension class implements `keepAlive`.

Reply:
Fixed